src/modules/productData.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+const getData = async () => {
+ 
+  const response = await axios.get("/products", {
+  });
+
+  return response;
```

Comment:
so here it will say `response.data.products`, then you are only returning the array that you want to your component

Reply:
This should be fixed now!